README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Ui](docs/images/Ui.png)
 1. Main Screen - add new flashcards or decks, edit existing flashcards or decks
-2. Review Screen - study a particular deck of cards
+2. Review Screen - study a particular masterDeck of cards
```

Comment:
All the occurences of decks are modified to masterDeck in Developer Guide and User Guide :') Perhaps you can copy paste from our team repo's Developer Guide and User Guide again @HJ-PRK